awx/ui_next/src/components/AddRole/AddResourceRole.jsx

```diff
@@ -76,22 +77,58 @@ function AddResourceRole({ onSave, onClose, roles, resource, onError }) {
     setCurrentStepId(step.id);
   };
 
+  const userOrTeamAndCredentialShareOrg = async (userOrTeam, role) => {
```

Comment:
would love help on rename of `userOrTeam` as it doesn't roll off the tongue. I thought about `userUnit` but thought that didn't make much sense either.....suggestions?

Reply:
@AlexSCorey just call it something like `sharesCredentialOrg` and let the additional context be described by the variable you call it with.

`sharesCredentialOrg(user)`
`sharesCredentialOrg(team)`  is plenty descriptive